RadixWallet/Clients/DeepLinkHandlerClient/DeepLinkHandlerClient+Interface.swift

```diff
@@ -0,0 +1,21 @@
+// MARK: - DeepLinkHandlerClient
+/// A client that manages the handling of the received deepLink.
+/// It will delegate the deepLink to the respective subcomponent.
+public struct DeepLinkHandlerClient: DependencyKey {
+	public var handleDeepLink: HandleDeepLink
+	public var setDeepLink: AddDeepLink
+	public var hasDeepLink: HasDeepLink
+}
+
+extension DeepLinkHandlerClient {
+	public typealias HandleDeepLink = () -> Void
```

Comment:
`@Sendable` on all function types?

Reply:
Not needed, as those functions are only called from the main thread.